### PR TITLE
Use JLL package

### PR DESCRIPTION
**Pull request by helgee**
@giordano Do you think it's fine to just drop pre-1.3 Julia versions here? There are no new features or bug fixes and the old versions will remain functional.

### CODE REVIEW

**Comment by giordano** [comment]
Yes, I'm not afraid of removing support for old versions of Julia: they won't enjoy new releases of the package, but the old ones will of course work. I believe the much easier installation and lower maintenance are worth the breakage.

Note that cirrus CI is probably kind of broken. You can just drop it and add `freebsd` to the operating systems for Travis

**Review by giordano — APPROVED** [review]

File: Project.toml [hunk 1]
```diff
@@ -1,14 +1,15 @@
 name = "SPICE"
 uuid = "5bab7191-041a-5c2e-a744-024b9c3a5062"
-version = "0.1.1"
+version = "0.2.0"
 
 [deps]
-BinaryProvider = "b99e7846-7c00-51b0-8f62-c81ae34c0232"
+CSPICE_jll = "07f52509-e9d9-513c-a20d-3b911885bf96"
 Libdl = "8f399da3-3557-5675-b5ff-fb832c97cbdb"
```
Comment (giordano) [inline Project.toml]: I think you can remove `Libdl`

**Comment by helgee** [comment]
In principle it would suffice to test on 1.5 and nightly, right?

**Comment by giordano** [comment]
I usually test packages with the oldest supported version of Julia (`1.3` here) and the latest one (`1`, which is an alias for the latest stable release in the v1 series)

**Comment by giordano** [comment]
...in addition to nightly

**Pull request merged**